### PR TITLE
feat(schedule_delete): implement DELETE for /schedule

### LINKED ISSUE #88: [FEATURE] Implement DELETE for /schedule
## Description

Implement endpoint in the backend according to the Swagger definition (when it's done)
- [ ] implement REST API handler 
- [ ] add database function
- [ ] add unit tests
- [ ] add integration tests

### PR DESCRIPTION
**Pull request by cornhubius**
**What**

- added handler for /schedule delete
- added db function
- added test for them

closed #88

### CODE REVIEW

File: backend/server/schedule_delete.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+schedule_delete = Blueprint('schedule_delete', __name__)
+
+
+@schedule_delete.route("/schedule/interval_id=<interval_id>", methods=['DELETE'])
+def delete_slots(interval_id):
+    """ Delete slots by id
+    """
+    delete_slot = delete_slot_by_id_for_schedule(interval_id)
```
Comment (mentortechabc) [inline backend/server/schedule_delete.py:12]: how do you process a missing interval_id? (error code, tests)

**Review by mentortechabc — CHANGES_REQUESTED** [review]

File: backend/server/schedule_delete.py (hunk 1 above)
Comment (cornhubius) [reply backend/server/schedule_delete.py:12]: there is a check in the `backend/tools/delete_slot_by_id_for_schedule.py` file
```py
def delete_slot_by_id_for_schedule(interval_id):
    try:
        models.db.session.delete(Slots.query.get(interval_id))
        models.db.session.commit()
        return {
            'msg': 'successfully deleted'
        }
    except Exception:
        models.db.session.rollback()
        return {
            'msg': 'delete error'
        }
```

Comment (cornhubius) [reply backend/server/schedule_delete.py:12]: and test ```py
def test_delete_schedule_slot_if_not_id():
    create_test_app_with_db()
    assert delete_slot_by_id_for_schedule(1) == {'msg': 'delete error'}
```

Comment (DramaTourG) [reply backend/server/schedule_delete.py:12]: response.status will be 200 even if an error occurred

**Review by mentortechabc — APPROVED** [review]

**Pull request merged**